config.js

```diff
@@ -10,34 +20,37 @@ export const cointypes = {
 }
 
 /**
- * Lock time, minute
+ * Lock time (in seconds).
  */
-export const LOCK_TIME = 24 * 60
+export const LOCK_TIME = 5 * 60
```

Comment:
Time is not in seconds. Did you mean to change https://github.com/oasisprotocol/oasis-wallet-ext/blob/1922c520330c94a36fce9b542efd236c43988ea6/src/background/service/APIService.js#L76-L90

Reply:
Fixed in https://github.com/oasisprotocol/oasis-wallet-ext/pull/133

Reply:
Oh, thanks for catching this @lukaw3d :slightly_smiling_face: !